libpurecoollink/dyson.py

```diff
@@ -676,11 +725,13 @@ def __init__(self, payload):
         """
         json_message = json.loads(payload)
         data = json_message['data']
-        self._humidity = int(self.__get_field_value(data, 'hact'))
+        humidity = self.__get_field_value(data, 'hact')
+        self._humidity = 0 if humidity == 'OFF' else int()
         volatil_copounds = self.__get_field_value(data, 'vact')
         self._volatil_compounds = 0 if volatil_copounds == 'INIT' else int(
             volatil_copounds)
-        self._temperature = float(self.__get_field_value(data, 'tact'))/10
```

Comment:
Exception was thrown here due to `self.__get_field_value(data, 'tact')` returned 'OFF'